### PR TITLE
TASK-54167: wallet creation choices

### PR DESCRIPTION
**Pull request by GouadriaHanen**
add new welcome screen for wallet creation allows to choose either to create an internal wallet or connect with metamask wallet

### CODE REVIEW

File: wallet-services/src/main/resources/locale/addon/Wallet_en.properties [hunk 1]
```diff
@@ -451,4 +451,14 @@ exoplatform.wallet.button.addNewPool=Add a new pool
...
\ No newline at end of file
+exoplatform.wallet.warn.selectedAddress=Erroneous account, please select the corresponding one
+
+exoplatform.wallet.label.meedsBtnTitle= create Meeds wallet
+exoplatform.wallet.label.internalWallet=Meeds wallet
+exoplatform.wallet.label.createInternalWallet=Ideal for getting started quickly, your Meeds wallet is protected by a password that only you know
+exoplatform.wallet.label.metamask=MetaMask
+exoplatform.wallet.label.createMetamaskWallet=You already have a Metamask account. You can connect it to eXo. Your wallet will then be available and managed directly in Metamask.
+exoplatform.wallet.label.createMetamaskWalletLearnMore=Learn more:
+exoplatform.wallet.metamask.welcomeMessage=Welcome to digital workplace \n\nClick to SignIn and accept digital workplace terms. This actions will not trigger a blockchain transaction and or any Gas fees. \n\nWallet address: \n{0} \n\nNonce: \n{1}
```
Comment (ahamdi) [inline wallet-services/src/main/resources/locale/addon/Wallet_en.properties]: We may use the same pattern, WDYT ? 
```suggestion
exoplatform.wallet.label.metamask.welcomeMessage=Welcome to digital workplace \n\nClick to SignIn and accept digital workplace terms. This actions will not trigger a blockchain transaction and or any Gas fees. \n\nWallet address: \n{0} \n\nNonce: \n{1}
```

Comment (GouadriaHanen) [reply wallet-services/src/main/resources/locale/addon/Wallet_en.properties]: done!

File: qqq [hunk 2]
```diff
@@ -0,0 +1,4297 @@
+[33mcommit 30f4b0ef63e862682c86c6c059e1814ec3090363[m[33m ([m[1;36mHEAD -> [m[1;32mfeature/meedsv2[m[33m)[m
```
Comment (ahamdi) [inline qqq]: We need to remove this file

Comment (GouadriaHanen) [reply qqq]: done!

**Review by ahamdi — APPROVED** [review]

**Pull request merged**